proposals/intptr-operators.md

```diff
@@ -0,0 +1,54 @@
+# Operators should be exposed for `System.IntPtr` and `System.UIntPtr`
+
+* [x] Proposed
+* [ ] Prototype: [Complete](https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)
+* [ ] Implementation: [In Progress](https://github.com/dotnet/roslyn/BRANCH_NAME)
+* [ ] Specification: [Not Started](pr/1)
```

Comment:
I've changed them all to `Not Started` and dropped the link for now. We can adjust as appropriate later.